scripts/ci/test_static.sh

```diff
@@ -51,7 +51,6 @@ run_style azure.cli.command_modules.configure
 #run_style azure.cli.command_modules.cosmosdb
 #run_style azure.cli.command_modules.dla
 #run_style azure.cli.command_modules.dls
-#run_style azure.cli.command_modules.eventgrid
```

Comment:
This is a whitelist, so you need to uncomment this rather than remove it.

Reply:
done